<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manager</class>
 <widget class="QMainWindow" name="Manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="treePlugin">
       <attribute name="headerVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="headerCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>70</number>
       </attribute>
       <attribute name="headerHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="headerMinimumSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="headerShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="headerStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Plugin</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Path</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="createButton">
         <property name="text">
          <string>CreatePlugin</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="subscribeButton">
         <property name="text">
          <string>Subscribe</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="target_id"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="block_name"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
